sdk/src/main/java/io/dapr/actors/ActorProxyToAppHttpAsyncClient.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.*;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.UUID;
+
+/**
+ * Http client to call actors methods.
+ */
+class ActorProxyToAppHttpAsyncClient extends DaprClientBase implements ActorProxyToAppAsyncClient {
```

Comment:
This can move to io.dapr.actors.client package. Fine for now, it can be moved when ActorProxy is added.